### PR TITLE
Add tutorial on how to add new entity

### PR DESCRIPTION
**Pull request by xtuchyna**
## This introduces a breaking change

- [ ] Yes
- [X] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,14 +80,24 @@ For each repository is possible to obtain the following plots:
...
+* method ```analyse_entity``` should be then called in ```bot_knowledge.analyse_projects``` with your new entity enum as a parameter
+* if you created Issue from the first point, reference it in your Pull Request
 
-.. code-block:: console
+Whenever you are not sure what to do, always try to be inspired by the current implementations of analysis methods or feel free to create new Issue with question you are dealing with.
```
Comment (pacospace) [inline README.md:93]: Same here on general rules on how to contribute, rather than in a specific section of the README file.

File: README.md [hunk 2]
```diff
@@ -80,14 +80,24 @@ For each repository is possible to obtain the following plots:
 Add new entity types to be stored
 ====================================
 
-TBD
+If you want to contribute by adding new entity that will be analysed from GitHub repository and stored as a knowledge, here is the acceptance criteria that should be satisfied for the implementation to work with the SrcOpsMetrics project:
 
-Usage - Reviewer Reccomender
-============================
+* if you are not part of the thoth-station or AICoE organization, open new Issue and engage in conversation why do you think this new entity should be analysed and stored and what are the benefits of doing so according to the goal of SrcOpsMetrics project.
```
[the remark this answers is not in the document]
Comment (xtuchyna) [reply README.md:85]: hmm yes that is actually very reasonable

[the remark this answers is not in the document]
Comment (xtuchyna) [reply README.md]: WDYT now, can it be?

File: README.md [hunk 3]
```diff
@@ -77,17 +77,24 @@ For each repository is possible to obtain the following plots:
...
 Usage - Reviewer Reccomender
 ============================
-
-.. code-block:: console
-
-    PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -r True
+KnowledgeStorage
+.. code-block:: consoleKnowledgeStorage
+KnowledgeStorage
+    PYTHONPATH=. pipenv run srcoKnowledgeStoragepsmetrics/cli.py --project <project_name> -r True
```
Comment (pacospace) [inline README.md:97]: why these changes?

Comment (xtuchyna) [reply README.md]: yuck, i don't even now how these got here...
fixed now!
